ts/Series/Bubble/BubbleSeries.ts

```diff
@@ -528,32 +552,45 @@ class BubbleSeries extends ScatterSeries {
      * @private
      */
     public animate(init?: boolean): void {
-        if (
-            !init &&
-            this.points.length < (this.options.animationLimit as any) // #8099
-        ) {
-            this.points.forEach(function (point): void {
-                const { graphic } = point;
-
-                if (graphic && graphic.width) { // URL symbols don't have width
-
-                    // Start values
-                    if (!this.hasRendered) {
-                        graphic.attr({
-                            x: point.plotX,
-                            y: point.plotY,
-                            width: 1,
-                            height: 1
-                        });
-                    }
+        const animationLimit = pick(this.options.animationLimit, Infinity);
 
-                    // Run animation
-                    graphic.animate(
-                        this.markerAttribs(point),
-                        this.options.animation
-                    );
-                }
-            }, this);
+        if (!init && this.points.length < animationLimit) { // #8099
+            const blendColors = this.options.blendColors;
+
+            // Why ['']? If standard single bubble, perform animation only for
+            // one point (one graphic). String element matches in TS
+            (blendColors || ['']).forEach((_, i: number): void => {
```

Comment:
It looks a bit weird to use a `forEach` loop just to get the index. Maybe better with a regular `for` loop?

`for (let i = 0; i < blendColors.length || 1; i++) {`